interfaces/builtin/locale_control.go

```diff
@@ -70,6 +70,7 @@ func init() {
 		name:                  "locale-control",
 		summary:               localeControlSummary,
 		implicitOnClassic:     true,
+		implicitOnCore:        true,
```

Comment:
No tests checking this? Are we missing tests that exercise core?

Reply:
I added a small unit test to check this on the other PR.